examples/bond/assets/bond-dapp-stateful.py

```diff
@@ -0,0 +1,209 @@
+from pyteal import *
+
+def approval_program():
+    """
+    A stateful smart contract which will store the bond parameters:
+    issue price, nominal price, maturity date and coupon value, total amount.
+    Additionally, the App will store a reference to the currently
+    tradeable bond ASA and the current epoch number.
+    """
+
+    store_manager = Bytes("store_manager")
+    issue_price = Bytes("issue_price")
+    nominal_price = Bytes("nominal_price")
+    maturity_date = Bytes("maturity_date")
+    coupon_value = Bytes("coupon_value")
+    issuer_address = Bytes("issuer_address")
+    epoch = Bytes("epoch")
+    total = Bytes("total")
+    current_bond = Bytes("current_bond")
+    max_amount = Bytes("max_amount")
+    creator = Bytes("creator")
+
+    # verify rekey_to and close_rem_to are set as zero_address
+    basic_checks = And(
+        # Always verify that the RekeyTo property of any transaction is set to the ZeroAddress
+        # unless the contract is specifically involved ina rekeying operation.
+        Txn.rekey_to() == Global.zero_address(),
+        Txn.close_remainder_to() == Global.zero_address(),
+        Txn.asset_close_to() == Global.zero_address()
+    )
+
+    # initialize variables
+    on_initialize = Seq([
+        App.globalPut(store_manager, Txn.application_args[0]),
```

Comment:
Is this the bond-token manager or bond-dapp manager? Could you please add some documentation regarding the keys in global state not present in readme? Currently it is not clear

Reply:
this is store manager as the name suggests, `store manager`. will add from spec document